images/runtime/node/template.Dockerfile

```diff
@@ -1,5 +1,6 @@
+ARG RUNIMAGE_BASE
 # Startup script generator
-FROM golang:1.11-stretch as startupCmdGen
+FROM golang:1.14.2-{RUNIMAGE_BASE} as startupCmdGen
```

Comment:
>RUNIMAGE_BASE [](start = 20, length = 13)

same here and other similar places...this variable is a bit confusing...how about saying DEBIAN_FLAVOR?